### PR TITLE
chore: add prettier to format code

### LINKED ISSUE #189: [Style] How to format code
# Question

I found that I can't format code when I am working for #188. I try to format code(my default formatter is `prettier`), one of the results is that single quotes are turned into double quotes. So, I want to know the way of formating code.  Thanks !

# Link

https://github.com/user-attachments/assets/76440088-54e6-4cdb-9ddb-77d91edeab07

### PR DESCRIPTION
**Pull request by chouchouji**
# Issue

close #189 

# Link

image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2617,6 +2617,11 @@ prettier@^2.7.1:
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.8.8.tgz#e8c5d7e98a4305ffe3de2e1fc4aca1a71c28b1da"
   integrity sha512-tdN8qQGvNjw4CHbY+XXk0JgCXn9QiF21a55rBe5LJAU+kDyC4WQn4+awm2Xfk2lQMk5fKup9XgzTZtGkjBdP9Q==
 
+prettier@^3.3.3:
```
Comment (iosh) [inline yarn.lock:2620]: Here can use yarnpkg.com npm source

Comment (chouchouji) [reply yarn.lock:2620]: Can you tell me how to do it? I executed `yarn add prettier -D`

Comment (chouchouji) [reply yarn.lock:2620]: I changed it from 'https://registry.npmmirror.com/' to 'https://registry.yarnpkg.com/'. Please review again. Thanks.

**Review by iosh — APPROVED** [review]

**Comment by iosh** [comment]
Thanks.

**Pull request merged**